tests/SessionTest.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System;
 using AeroGear.OAuth2;
 using Microsoft.VisualStudio.TestPlatform.UnitTestFramework;
```

Comment:
does this have benefits over nunit? Or would this MS framework be also compatible with Mono? just wondering :-)